apluslms_roman/cli.py

```diff
@@ -334,11 +341,18 @@ def get_config(context):
         if context.args.project_config:
             project_config = abspath(expanduser(expandvars(context.args.project_config)))
             return CourseConfig.load_from(project_config)
-        return CourseConfig.find_from(getcwd())
+        try:
+            return CourseConfig.find_from(getcwd())
+        # getcwd() is always a valid path, so only the lack of a roman config
+        # file can cause a CourseConfigError
+        except CourseConfigError as err:
+            raise FileNotFoundError(str(err) +
```

Comment:
init_action catches this, so exit can't be used because even is SystemExit is catched, it prints the error.

Reply:
We need to revisit the design for interfaces of ProjectConfig, get_config and init_action. I think we need to find a solution where get_config and init_action are separate, but still simple (e.g. no code duplication).

For example, I think we need FileNotFoundError in ProjectConfig, so we can see when missing file is the problem and when the file itself is the problem.